collapsiblecalendarview2/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,8 @@
     <string name="friday">FRI</string>
     <string name="saturday">SAT</string>
     <string name="sunday">SUN</string>
+    <string name="content_description_ll_calendar_expand">Expand calendar</string>
+    <string name="content_description_ll_calendar_collapse">Collapse calendar</string>
```

Comment:
Are these strings complete?

Reply:
What does that mean? What do you think they should be?

Reply:
I made these strings into two like you said and programmatically made the content description one of them depending on if the calendar is expanded or collapsed.